README.md

```diff
@@ -33,9 +33,9 @@ npm install
 
         npm install
 
-3.  Tag a new version by running the following command: `npm version [ major | minor | patch ]`. Choose `major`, `minor`, or `patch` depending on the kind of update according to [Semantic Versioning](https://semver.org) rules.
+3.  Tag a new version by running the following command: `npm version [ major | minor | patch ]`. Choose `major`, `minor`, or `patch` depending on the kind of update according to [Semantic Versioning](https://semver.org) rules. Updates are categorized as major, minor, or patch, depending on the type of change. A major change is defined as any modification that breaks compatibility with the core schema, such as a change in a property or the addition of a new property that affects the mission pipelines. Introducing a new mission and new core schema constitutes a minor change, while modifying the new mission schema and adding a new property to the core schema are considered patches.
```

Comment:
This just looks like a summary of SemVer. What is the value of adding it?

Reply:
This includes examples and we prefer documents to be self-consistent for clarity. 

@jracusin Do you also think it's not adding any value?

Reply:
Alright. Some corrections:

- Major revisions are for changes that break compatibility; they are not necessarily limited to core schema changes.
- Minor revisions are for introduction of new features: new schema, or new fields in existing schema.
- Patch revisions are for bug fixes that do not break compatibility.

Reply:
Can you also include a note clarifying that we will run these steps, and that producers should ignore it? I have seen a couple times now of new producers running the version command in their PRs

Reply:
Yes, thanks. I will add a note here and will add a line in Notices producers document as well. I have also seen this confusion.

Reply:
> Can you also include a note clarifying that we will run these steps, and that producers should ignore it? I have seen a couple times now of new producers running the version command in their PRs

Done, see below screenshot.